src/components/AppCard/AppCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+// import styles from './AppCardEdit.module.css';
+import styles from './AppCard.module.css';
+
+import { EditCard } from './edit/editCard';
+import { NormalCard } from './normal/normalCard';
+
+export const AppCard = ({ front, back }) => {
+	const [side, setSide] = useState(true);
+	const [isEdit, setEdit] = useState(false);
+	return (
+		<>
+			<div>
+				{!isEdit && (
+					<NormalCard
+						front={front}
+						back={back}
+						side={side}
```

Comment:
**nitpick:** This formatting is really hard to read - let's try to fix prettier in your project